src/platform_impl/linux/window.rs

```diff
@@ -242,10 +242,6 @@ impl Window {
       }
     }
 
-    if let Parent::ChildOf(parent) = pl_attribs.parent {
-      window.set_transient_for(Some(&parent));
-    }
-
```

Comment:
why remove this?

Reply:
i thought it was part of your workaround, it's not needed for mine at least

Reply:
no that's a different feature. I just reordered it.

Reply:
so why is it in this PR?

Reply:
I just moved it up by mistake while working on the workaround. Should be good now.

Reply:
ahh i understood now, alright